packages/inter-protocol/src/proposals/startPSM.js

```diff
@@ -170,6 +271,23 @@ export const startPSM = async (
     E(governorFacets.creatorFacet).getAdminFacet(),
   ]);
 
+  /** @param {MetricsNotification} metrics */
+  const restoreMetrics = async metrics => {
+    const anchorBalancePayments = await anchorBalancePaymentsP;
+    const anchorPmt = anchorBalancePayments.get(anchorBrand);
+
+    const { anchorPoolBalance: _a, ...nonPaymentMetrics } = metrics;
+
+    const seat = E(zoe).offer(
+      E(psmCreatorFacet).makeRestoreMetricsInvitation(),
+      harden({ give: { Anchor: metrics.anchorPoolBalance } }),
+      harden({ Anchor: anchorPmt }),
+      harden(nonPaymentMetrics),
+    );
+    await E(seat).getPayouts();
+  };
+  await (oldState.metrics && restoreMetrics(oldState.metrics));
```

Comment:
why await first? we generally pass the promise in,
```suggestion
  await restoreMetrics(oldState.metrics);
```
it's an async function so it can do the awaitiing,
```
const metrics = await metricsP;
```

Reply:
The idea is to `restoreMetrics` only if there are some `oldState.metrics` to restore, avoiding the expensive `E(zoe).offer(...)` otherwise.

`oldState.metrics` is not a promise.

Reply:
oic. normally I'd expect,
```js
if (oldState.metrics) {
  await restoreMetrics(oldState.metrics));
}
```
but I suppose that trips on the jessie awaits rule